### PR TITLE
added line between steps in prototype stepper

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* /bcgov/entity#1892

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/common/Stepper.vue [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
-  <v-container id="step-buttons-container" class="list-item justify-space-around py-8">
+  <v-container id="step-buttons-container">
     <template v-for="(step, index) in steps">
-      <div class="text-center align-self-start" v-bind:key="index">
-        <v-btn :id=step.id outlined fab color="primary" :to=step.to :disabled=step.disabled>
-          <v-icon>{{step.icon}}</v-icon>
-        </v-btn>
-        <v-icon size="30" color="green darken-1" v-show=step.valid>mdi-check-circle</v-icon>
-        <div class="pre-line">{{step.text}}</div>
+      <div class="step" v-bind:key="index">
```
Comment (cameron-freshworks) [inline src/components/common/Stepper.vue]: Not sure you 'have' too put v-bind before the `:key` but thats up to you!

File: src/components/common/Stepper.vue [hunk 2]
```diff
@@ -177,14 +182,58 @@ export default class Stepper extends Vue {
...
+  position: relative;
+  width: 100%;
+  text-align: center;
+}
+
+.step__line {
+  position: absolute;
+  top: 50%;
+  left: 0;
+  height: 2px;
```
Comment (cameron-freshworks) [inline src/components/common/Stepper.vue]: Just curious why we are using px here and rem in other spots?

File: src/components/common/Stepper.vue [hunk 3]
```diff
@@ -177,14 +182,58 @@ export default class Stepper extends Vue {
...
   position: relative;
   margin-top: -32px;
   margin-left: -16px;
   background: $BCgovInputBG;
   border-radius: 50%;
+  z-index: 3;
+}
+
+.step__label {
+  margin-top: 10px;
```
Comment (cameron-freshworks) [inline src/components/common/Stepper.vue]: Same q as above!

File: src/components/common/Stepper.vue (hunk 1 above)
Comment (severinbeauvais) [reply src/components/common/Stepper.vue]: You're right, not required. But hey it's a "prototype".

File: src/components/common/Stepper.vue (hunk 2 above)
Comment (severinbeauvais) [reply src/components/common/Stepper.vue]: This is the height of the connector line. It never needs to be scaled, which is the usual reason to use "root ems". 1 px seems too thin to me and 3 px seems too thick.

File: src/components/common/Stepper.vue (hunk 3 above)
Comment (severinbeauvais) [reply src/components/common/Stepper.vue]: I guess I didn't have a good reason to use rems. This just seemed like a good value. It's also easier to read than 0.625rem.
To keep the number of font sizes to a minimum I use a small set of rem values for fonts. And I use a small set of values for margins and padding. But here I'm just tweaking the space between elements so px is more specific (and there's no need to scale it).

File: src/components/common/Stepper.vue (hunk 1 above)
Comment (severinbeauvais) [reply src/components/common/Stepper.vue]: Ok, I didn't see this earlier and remove it...

Comment (cameron-freshworks) [reply src/components/common/Stepper.vue]: right on!

File: src/components/common/Stepper.vue (hunk 3 above)
Comment (cameron-freshworks) [reply src/components/common/Stepper.vue]: Sounds good, was just curious!

**Pull request merged**